plugins/source/aws/client/spec.go

```diff
@@ -60,6 +61,7 @@ type Spec struct {
 	TableOptions              *tableoptions.TableOptions `json:"table_options,omitempty"`
 	Concurrency               int                        `json:"concurrency"`
 	EventBasedSync            *EventBasedSync            `json:"event_based_sync,omitempty"`
+	Scheduler                 scheduler.Strategy         `json:"scheduler,omitempty"`
```

Comment:
The default is DFS (it's an enum with zero value) so we don't need to set a default explicitly